### PR TITLE
feat: Implemented amazon login page

### PR DESCRIPTION
**Pull request by FayazCodeCraft**
- Created amazon login page using html and css

### CODE REVIEW

File: login/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
...k7KdqWFSCTg5K9S2qT08Jtl4dzXs/bckPCkb18iKOFMmSwIvg2p4/nTq8SQ9wmyjRKBNGK9Gr5CNimCfVrBULFuRJ648oPND4JJwfGsb75/o5OPUXyebTfrwHncYyZblQ9rIkfWBd8q0T3HlnxyNbBwIbXdZQYrkWdkSpwmHmvS644SnaKVbb+XGYgfClYbeIPFC0Dsi2ZpxtvGmSxUhv0K2UnSVTuT2Xm1NZ+RToE6oVrSdI5KNt1r1ohSQD+bPysZTrhLkUS/UqhSdqiJcWVirUlzxaI5GNoujrJdSyYiflS08QVXO4hdDRpiCMT2pZLnBPddFH0cjm5J4l5YmYbrFlypp4kVulyTsV0PZurVWrStli7Z3keGIZLOiQ9LAFhq1RFCPHq76UjZnEKlWFJN86NKC18i7VjY43br5RplVykWi9XlfcjyyKfYZrVDdc0e8oh3R/mGglst8SDJEayjKRLCtC5oUvlMgCSsFSxTMAR30z91En26HX5CM+b1T+Sx8jVsVXbTpWk40S18V5GdQHBvjKn+7C784WcUA3oCJlhUbQxe7ZxhVNruJr35TbzKZ7vVdbXYyRvzuW3TBx4isaP7uD4bM/Grha6bmXy4P+YvUe3m48N7jNr+ezfQA0zRn8w8e0iEwn+nLt3utZ/7l5uLm4tY0tdndx4/q5zPxZ9py+vJ9j+P5pr7+yOEcDN5C17Wbydsdp8Jna2reR4/oQLgSi9Ttnfcmp8lyO1spnqYvPmdQB8D8Wjd13b+Yv3ayTlYLLcjRiTLX37M0HgvTv6auBcpt71cvztbp+mHhB+mpb4Jb/wrtfjHrTZA9Qjn/8ma19p64y5s/bHaa2Hfo25UwbPWLrxvjj2QuSkOgXCCdtlvcL1fzycTzpgIvOA7MV383l7vgoinU3XLxnc9+d7IRd7emEEVIZ4oqofm760vBbqdpvLnVKCOvlx573M9W3zriH8L6YqezcqF6EdIUpOLtKqocs19cD/bwVputmZAugUhBf/GQnJW/8oDwBN58udiGk1IifvjXN3dYyZ7FW98t7xfXW9/XfH+7vdxcBRXiuwd1MAQ1NODdJ1YAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADwK/kHm0hm4zyWkikAAAAASUVORK5CYII=" alt="Amazon Logo">
```
Comment (nikhilsnayak) [inline login/index.html:11]: This image URL looks ugly on markup. Can you use a better image URL?

File: login/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <div class="login-box">
+      <h2>Sign-In</h2>
+      <form>
+        <label for="email">Email or mobile phone number</label>
```
Comment (nikhilsnayak) [inline login/index.html:15]: Add only login through Email. I think we don't need phone number in this label

**Review by nikhilsnayak — CHANGES_REQUESTED** [review]
- Also can you add some styles to position the logo to top-left corner
- Can you add a button for signup

File: login/index.html (hunk 1 above)
Comment (vinusuhas-codecraft) [inline login/index.html:11]: store the image locally

Comment (FayazCodeCraft) [reply login/index.html]: Fixed.

Comment (FayazCodeCraft) [reply login/index.html]: Fixed

**Review by nikhilsnayak — APPROVED** [review]
I think all the changes looks good. we can merge it. Please stash all the commits into one meaningful commit to merge the PR

File: login/index.html (hunk 2 above)
Comment (FayazCodeCraft) [reply login/index.html]: Fixed

**Review by vinusuhas-codecraft — APPROVED** [review]

**Pull request merged**